assets/src/components/map.tsx

```diff
@@ -131,6 +131,7 @@ export const FullscreenControl = createControlComponent(
   Leaflet.control.fullscreen
 )
 
+// TODO: use tilesetForType instead
 const tilesetUrl = (): string => appData()?.tilesetUrl || ""
```

Comment:
For the order of operations of deploys, I **think** we'd want to hold off on making this change until after doing a first deploy with the new env vars configured

Reply:
Makes sense!